core/vgmgr/virtual_group_manager.go

```diff
@@ -41,6 +42,12 @@ type GVGPickFilter interface {
 	CheckGVG(gvgMeta *GlobalVirtualGroupMeta) bool
 }
 
+// GVGMetaChecker is used to check sp pick condition.
+type GVGMetaChecker interface {
+	// CheckGVG returns true when match pick request condition.
+	CheckGVG(migrateGVGUnitMeta []*spdb.MigrateGVGUnitMeta) bool
```

Comment:
The interfaces defined here are all used in base/gfspvgmr. The interface of GVGMetaChecker should not be placed here.